### PR TITLE
feat: add util to wait for transaction receipt

### LINKED ISSUE #24: Send task waits 2 seconds instead of waiting for ETH answer
Send task should wait for ethereum receipt

EthClient does not have  a method to wait for receipt.

EigenSDK has a method called WaitForTransactionReceipt in InstrumentedEthClient
But is needs telemetry to work

https://github.com/Layr-Labs/eigensdk-go/blob/master/chainio/clients/eth/instrumented_client.goreceipt := avsWriter.Client.WaitForTransactionReceipt(context.Background(), tx.Hash())

### PR DESCRIPTION
**Pull request by NicolasRampoldi**
- Added util in `core/utils/eth_client_utils.go` to wait for transaction receipt.

### CODE REVIEW

**Comment by MauroToscano** [comment]
We need to add this to the operator too

**Comment by NicolasRampoldi** [comment]
> We need to add this to the operator too

Perfect. When the operator is pushed to main I will add it there too 👍

**Review by JuArce — APPROVED** [review]

**Comment by MauroToscano** [comment]
> > We need to add this to the operator too
> 
> Perfect. When the operator is pushed to main I will add it there too 👍

I meant in the deposit into strategy

File: core/utils/eth_client_utils.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	"context"
+	"github.com/Layr-Labs/eigensdk-go/chainio/clients/eth"
+	common2 "github.com/ethereum/go-ethereum/common"
```
Comment (JuArce) [inline core/utils/eth_client_utils.go]: Import this as `gethcommon` for better readibiity

**Review by JuArce — CHANGES_REQUESTED** [review]

**Review by JuArce — APPROVED** [review]

**Review by taturosati — APPROVED** [review]
Create an issue to add max retries

**Pull request merged**